doc/introduction/interfaces.rst

```diff
@@ -76,6 +76,18 @@ See the links below for walkthroughs of each specific interface:
         </div>
     </div>
 
+In addition to the core interfaces discussed above, PennyLane also provides higher-level classes for
+converting QNodes into both Keras and ``torch.nn`` layers:
+
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+
+    pennylane.qnn.KerasLayer
+    pennylane.qnn.TorchLayer
+
+:html:`</div>`
```

Comment:
Nice!

Reply:
Was trying to work out the best way to make them 'front and center' without duplicating the details Tom has already added to the docstrings